static/app/components/replays/replayView.tsx

```diff
@@ -10,54 +9,48 @@ import ReplayPlayer from 'sentry/components/replays/replayPlayer';
 import space from 'sentry/styles/space';
 
 type Props = {
-  isFullscreen: boolean;
   toggleFullscreen: () => void;
 };
 
-function ReplayView({isFullscreen, toggleFullscreen}: Props) {
+function ReplayView({toggleFullscreen}: Props) {
   return (
     <Fragment>
       <ReplayCurrentUrl />
-      <PanelNoMargin isFullscreen={isFullscreen}>
-        <PanelHeader disablePadding noBorder>
+      <PlayerContainer>
+        <Panel>
           <ReplayPlayer />
-        </PanelHeader>
+        </Panel>
         <ScrubberMouseTracking>
           <PlayerScrubber />
         </ScrubberMouseTracking>
-      </PanelNoMargin>
-      <ReplayControllerWrapper>
-        <ReplayController toggleFullscreen={toggleFullscreen} />
-      </ReplayControllerWrapper>
+      </PlayerContainer>
+      <ReplayController toggleFullscreen={toggleFullscreen} />
     </Fragment>
   );
 }
 
-const ReplayControllerWrapper = styled(PanelBody)`
-  padding-top: ${space(1)};
-`;
+const PlayerContainer = styled('div')`
+  display: flex;
+  flex-direction: column;
+  flex-wrap: nowrap;
+  flex-grow: 1;
+  overflow: hidden;
 
-const PanelNoMargin = styled(Panel)<{isFullscreen: boolean}>`
```

Comment:
stopped using Panel here because a) was overriding a lot of stuff b) the scrubber nub was cutoff at the border

So if we add border and manage radius manually anyway, don't need this panel anymore. Replace with `const Panel = ...`